services/hexathons/src/routes/interaction.ts

```diff
@@ -105,6 +106,52 @@ interactionRoutes.route("/").post(
       });
     }
 
+    // If this is a check-in interaction, update the application status to CHECKED_IN
+    if (req.body.type === InteractionType.CHECK_IN) {
```

Comment:
we should also only allow members (i.e. hexlabs members) to check people in. see above code in file for example of how its done